src/bin/main.rs

```diff
@@ -63,6 +65,10 @@ async fn exec() -> Result<(), Error> {
     let args = WalletArgs::parse();
     let cmd = args.command.clone();
 
+    // set symbols to ASCII for Windows terminal compatibility
+    #[cfg(windows)]
+    symbols::set(symbols::ASCII);
```

Comment:
Consider using the fully qualified name. This will allow you to remove the import above and consequently the additional `#[cfg(windows)`].
```suggestion
    symbols::set(symbols::ASCII);
```

Reply:
This will also require the fully qualified name to be attached to `symbols::ASCII`:
```rust
requestty::symbols::set(requestty::symbols::ASCII);
```

Are you okay with it being specified twice?

Reply:
Maybe a scope is more appropriate?
```rust
#[cfg(windows)]
{
    use requestty::symbols::{self, ASCII};
    symbols::set(ASCII);
}
```

Reply:
As discussed offline, a scoped approach or a function based approach (as per your suggestion) would be a good future use if we see more differences appear in configuration options between Windows and Non-Windows builds. 

Given the size of this change, I'll opt for the latest change since it feels like a good trade off between explicitness and lines of code.